worker-markup-testing/src/main/java/com/hpe/caf/worker/markup/MarkupTestInput.java

```diff
@@ -68,4 +71,24 @@ public void setEmail(boolean isEmail)
     {
         this.isEmail = isEmail;
     }
+
+    /**
+     * Getter for property 'contentFieldName'.
+     *
+     * @return Value for property 'contentFieldName'.
+     */
+    public String getContentFieldName()
```

Comment:
This is the field to use in markup. 'getMarkupField' or 'getFieldToMarkup' would make more sense. Not a necessary change though.